gitlab-pipeline/stage/release.yml

```diff
@@ -53,11 +53,40 @@ release_docker_images:manual:
   extends: .template_release_docker_images
 
 
-release_docker_images:automatic:
-  only:
-    variables:
-      - $PUBLISH_RELEASE_AUTOMATIC == "true"
+release_docker_multiplatform_images:manual:
+  when: manual
   extends: .template_release_docker_images
+  # We only need init:workspace to inhering the environment - we retag upstream images from the same Git sha
+  dependencies:
+    - init:workspace
+  image: "registry.gitlab.com/northern.tech/mender/mender-test-containers:docker-multiplatform-buildx-v1-master"
+  script:
+    - for image in $($WORKSPACE/integration/extra/release_tool.py --list docker); do
+        if echo $image | grep -q -E 'mender-client|mender-qemu|mender-monitor|mender-gateway-qemu-commercial'; then
+            echo "Skipping client image $image";
+            continue;
+        fi;
+        # Target Docker tag
+        docker_url=$($WORKSPACE/integration/extra/release_tool.py --map-name docker $image docker_url);
+        version=$($WORKSPACE/integration/extra/release_tool.py --version-of $image --version-type docker --in-integration-version $INTEGRATION_REV);
+        # Source Docker tag
+        git_repo=$($WORKSPACE/integration/extra/release_tool.py --map-name docker $image git);
+        cicd_tag_base="$(echo ${version} | sed -r 's/(mender-[1-9]+\.[1-9]+\.).*/\1x/')";
+        cicd_tag_git="$(echo ${git_repo^^} | tr - _)_REV_GIT_SHA";
+        cicd_tag="${cicd_tag_base}_${!cicd_tag_git}";
+        # Do the actual copy
+        echo "Copying image ${docker_url}:${cicd_tag} into ${docker_url}:${version}";
+        regctl image copy ${docker_url}:${cicd_tag} ${docker_url}:${version};
+      done
```

Comment:
@oldgiova @kacf  Compared to #734, there were many small fixes to be done. Most notably:
* Skip client images
* Use `M.N.x_git` kind of tags for the source
* Use Docker Hub / Mender Registry (instead of GitLab registry) for the source

A dry run of the above scripts does something like:
```
Copying image registry.mender.io/mendersoftware/auditlogs:mender-3.7.x_cc93dc8e8dc20709ad58d2bba007c13197cc509f into registry.mender.io/mendersoftware/auditlogs:mender-3.7.4
regctl image copy registry.mender.io/mendersoftware/auditlogs:mender-3.7.x_cc93dc8e8dc20709ad58d2bba007c13197cc509f registry.mender.io/mendersoftware/auditlogs:mender-3.7.4
Copying image mendersoftware/create-artifact-worker:mender-3.7.x_dd672139045ddd3f226533296ae1f4736fb41896 into mendersoftware/create-artifact-worker:mender-3.7.4
regctl image copy mendersoftware/create-artifact-worker:mender-3.7.x_dd672139045ddd3f226533296ae1f4736fb41896 mendersoftware/create-artifact-worker:mender-3.7.4
Copying image mendersoftware/deployments:mender-3.7.x_342972f225f0e21b32e5dc2bea1948f44f55c68a into mendersoftware/deployments:mender-3.7.4
regctl image copy mendersoftware/deployments:mender-3.7.x_342972f225f0e21b32e5dc2bea1948f44f55c68a mendersoftware/deployments:mender-3.7.4
(...)
Copying image mendersoftware/iot-manager:mender-3.7.x_b8988417b6638177abdf34ea429d6ef73d22ce41 into mendersoftware/iot-manager:mender-3.7.4
regctl image copy mendersoftware/iot-manager:mender-3.7.x_b8988417b6638177abdf34ea429d6ef73d22ce41 mendersoftware/iot-manager:mender-3.7.4
Skipping client image mender-client-docker
Skipping client image mender-client-docker-addons
Skipping client image mender-client-qemu
Skipping client image mender-client-qemu-rofs
Copying image registry.mender.io/mendersoftware/mender-gateway:mender-3.7.x_8e00b84457e7a3d910db77e029af60907021a12b into registry.mender.io/mendersoftware/mender-gateway:mender-3.7.4
regctl image copy registry.mender.io/mendersoftware/mender-gateway:mender-3.7.x_8e00b84457e7a3d910db77e029af60907021a12b registry.mender.io/mendersoftware/mender-gateway:mender-3.7.4
Skipping client image mender-gateway-qemu-commercial
Skipping client image mender-monitor-qemu-commercial
Skipping client image mender-qemu-rofs-commercial
Copying image registry.mender.io/mendersoftware/tenantadm:mender-3.7.x_a153918c5d95e7d5873dfaef07ce4b8ed130e7f8 into registry.mender.io/mendersoftware/tenantadm:mender-3.7.4
regctl image copy registry.mender.io/mendersoftware/tenantadm:mender-3.7.x_a153918c5d95e7d5873dfaef07ce4b8ed130e7f8 registry.mender.io/mendersoftware/tenantadm:mender-3.7.4
Copying image mendersoftware/useradm:mender-3.7.x_40661aa7a2d63b5aa8ed24957f473fc5350286a1 into mendersoftware/useradm:mender-3.7.4
regctl image copy mendersoftware/useradm:mender-3.7.x_40661aa7a2d63b5aa8ed24957f473fc5350286a1 mendersoftware/useradm:mender-3.7.4
Copying image registry.mender.io/mendersoftware/useradm-enterprise:mender-3.7.x_80559da7366e2e450c6407a0d271506e2e1a0cf1 into registry.mender.io/mendersoftware/useradm-enterprise:mender-3.7.4
regctl image copy registry.mender.io/mendersoftware/useradm-enterprise:mender-3.7.x_80559da7366e2e450c6407a0d271506e2e1a0cf1 registry.mender.io/mendersoftware/useradm-enterprise:mender-3.7.4
(...)```